cypress/page/products-list.page.ts

```diff
@@ -2,11 +2,11 @@ class ProductListPage {
     private sauceLabsBolt: string;
 
     constructor() {
-        this.sauceLabsBolt = "#item_1_title_link";
+        this.sauceLabsBolt = ".inventory_list .inventory_item";
     }
 
-    public selectSauceLabsBolt(): void {
-        cy.get(this.sauceLabsBolt).click();
+    public findProductByName(): void {
+        cy.get(this.sauceLabsBolt).filter(':contains("Sauce Labs Bolt T-Shirt")').click();       
```

Comment:
You can receive the product name by paramenter

Reply:
change made

![image](https://user-images.githubusercontent.com/70541865/227645349-afade2ef-5aa4-4d4b-a475-6542a7094d86.png)